src/main/java/com/google/step/snippet/external/GeeksForGeeksClient.java

```diff
@@ -54,6 +59,27 @@ public Card search(String geeksLink) {
     String title = titles.first().text();
     String description = descriptions.first().text();
     String code = snippets.first().getElementsByClass(CODE_CLASS).text();
+    if (containsEscape(query.toLowerCase())
+        || containsEscape(geeksLink)
+        || containsEscape(title.toLowerCase())
+        || containsEscape(description.toLowerCase())) {
+      title = StringEscapeUtils.escapeHtml4(title);
+      description = StringEscapeUtils.escapeHtml4(description);
```

Comment:
Maybe we should only not clean and escape the code section and stick with cleaning title and description regardless. I doubt there will be any valuable HTML code that we need to display in the description or title.

Reply:
Agreed, except for the SO client which after Iris' PR will put code in the description